.github/workflows/tests.yml

```
@@ -13,8 +13,16 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        php: [7.2, 7.3, 7.4]
-        laravel: [^7.0]
+        php: [7.4, 7.3, 7.2.5]
```

```suggestion
        php: [7.2, 7.3, 7.4]
```
I think we should stick to PHP `7.2`